CondFormats/HcalObjects/interface/HcalMahiPulseOffsetsHost.h

```diff
@@ -0,0 +1,11 @@
+#ifndef CondFormats_HcalObjects_interface_HcalMahiPulseOffsetsPortable_h
+#define CondFormats_HcalObjects_interface_HcalMahiPulseOffsetsPortable_h
+
+#include "CondFormats/HcalObjects/interface/HcalMahiPulseOffsetsSoA.h"
+#include "DataFormats/Portable/interface/PortableHostCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
```

Comment:
```suggestion
```
Not needed ?

Reply:
Removed in 4615e8825352834ac1dd4f1bbe766b0cafc1d321